Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -574,34 +526,30 @@ - (NSURLSessionDataTask *)getShippingRatesForCheckout:(BUYCheckout *)checkout co
 
 #pragma mark - Payments
 
-- (NSURLSessionDataTask *)storeCreditCard:(BUYCreditCard *)creditCard checkout:(BUYCheckout *)checkout completion:(BUYDataCreditCardBlock)block
+- (NSURLSessionDataTask *)storeCreditCard:(BUYCreditCard *)creditCard checkout:(BUYCheckout *)checkout completion:(BUYDataCreditCardBlock)completion
 {
-	NSURLSessionDataTask *task = nil;
+	NSAssert(checkout, @"Failed to store credit card. No checkout provided.");
+	NSAssert(checkout.token, @"Failed to store credit card. No checkout token provided.");
```

Comment:
Maybe we need a macro that makes this assert simpler to include. This one is inconsistent with others.